From 1f36a8f997bf56f061a305cdf919db3227f5e58e Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Wed, 18 Jul 2012 10:49:53 +0200
Subject: [PATCH] pnmcrop detection

Simplification in the context of OpenCSW where the version of netpbm
is greater than 10 and netpbm is a prerequisite of building latex2html

The removed code is testing incorrectly the version greater than 8...
---
 config/config.pl |   20 +++-----------------
 1 file changed, 3 insertions(+), 17 deletions(-)

diff --git a/config/config.pl b/config/config.pl
index 1b243d7..be1935a 100755
--- a/config/config.pl
+++ b/config/config.pl
@@ -1281,26 +1281,12 @@ if($opt{'have_pstoimg'}) {
 	&checking('if pnmcrop can crop from one direction');
         my $timg = "config${dd}timg.pnm";
 	($stat,$msg,$err) = &get_out_err("$pnmcrop -l $timg");
-    } elsif ($vers > 8) {
+    } else {
 	my $sub_vers = '';
-	if ($vers =~ /9\.(\d+)/) {
-	    $sub_vers = $1;
-	    unless ($sub_vers > 11) {
-		$newcfg{'PNMBLACK'} = ' -black ';
-		print
-	"\n Please update to Netpbm 9.12+, from sourceforge.org/projects/netpbm/\n",
-	" else colored cropping-bars will not be removed.\n";
-	    } else {
-#		$pnmcrop .= ' -sides ';
-		$newcfg{'PNMCROPOPT'} = ' -sides ';
-	    }
-	} else { $newcfg{'PNMCROPOPT'} = ' -sides '; }
-	$pnmcrop .= ' -verbose ' if ($vers >= 10 || $sub_vers > 10);
+	$newcfg{'PNMCROPOPT'} = ' -sides ';
+	$pnmcrop .= ' -verbose ';
 	print "\n$pnmcrop";
 	$msg = 'there is nothing to crop'; $stat = '';
-    } else {
-	print "\nThis $vers for $pnmcrop is not recognisable.";
-	$stat = 1;
     }
 
     unless(!$stat && $msg =~ /^p\d+[\s\n]+\d+\s+\d+|nothing to crop/is) {
-- 
1.7.10.3

